node/src/main/scala/org/bitcoins/node/models/ColumnMappers.scala

```diff
@@ -42,9 +42,13 @@ trait ColumnMappers {
 
   implicit val uint64Mapper: BaseColumnType[UInt64] = {
     MappedColumnType.base[UInt64, BigDecimal](
-      { u64: UInt64 => BigDecimal(u64.toBigInt.bigInteger) } ,
+      { u64: UInt64 =>
+        BigDecimal(u64.toBigInt.bigInteger)
+      },
       //this has the potential to throw
```

Comment:
Would it better to do something like `UInt64(bigDec.toBigIntExact().getOrElse(throw new IllegalArgumentException("Could not map bigdecimal to UInt64")` to give a more meaningful error message?